packages/ndla-ui/src/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import { Cross } from '@ndla/icons/action';
+import { useTranslation } from 'react-i18next';
+const CloseButtonWrapper = styled.div`
+  position: relative;
+  top: 1px;
+  right: 1px;
+`;
+
+const CloseBut = styled.button`
+  background-color: transparent;
+  border: none;
+  display: flex;
+`;
```

Comment:
Foreslår å bytte navn til `StyledButton` feks. Knapper har ikke `cursor: pointer` som standard. Dette må settes for å få pekeren til å endre ikon til pointer (👆) ved hover.